src/js/components/glossary/search/ResultItem.jsx

```diff
@@ -89,6 +90,10 @@ export default class ResultItem extends React.Component {
 
     clickedLink() {
         this.props.selectTerm(this.props.item);
+        Analytics.event({
+            category: 'Glossary',
+            action: `Clicked ${this.props.item}`
```

Comment:
I console.logged this.props.item and it's not the Glossary term.  It's a Map object.  So you can try to pass the term to the clickedLink function and then use that or see if you can get the term from the object

Reply:
Okay. I did get the term but I did it with innerText. Find the passed term, then term.target.innerText